resources/assets/js/sidebar.js

```diff
@@ -0,0 +1,36 @@
+var $ = require('jquery');
```

Comment:
You'd probably want `jQuery` globally.

Getting back at my previous comment. You'd be better of using Vue here